.github/workflows/job_gpu_tests.yml

```diff
@@ -30,12 +34,14 @@ permissions: read-all
 
 jobs:
   GPU:
-    timeout-minutes: 80
-    runs-on: ${{ fromJSON(inputs.runner) }}
+    timeout-minutes: 120
```

Comment:
This is suspiciously long; internally the same tests take ~30-40 min on average. Will need to check if the test scope is valid (only smoke test cases & the compare the number of tests with their number in ie-tests-linux-ubuntu22-gpu Jenkins checks); and HW issues that could affect (e.g. over-heating)